src/plugins/vis_type_vega/public/data_model/vega_parser.ts

```diff
@@ -536,7 +583,7 @@ export class VegaParser {
    * @param {string} [key] field name of the current object
    * @private
    */
-  _findObjectDataUrls(obj, onFind, key) {
+  _findObjectDataUrls(obj: any, onFind: (data: Data) => void, key?: any) {
```

Comment:
Some here - can we make this an `unknown`?

Reply:
👍

Reply:
By converting it to `unknown` we can't access its properties so I'm having trouble finding a suitable type here. Can I have a help here?

Reply:
Reading through the code, this function recursively iterates the spec tree to find keys named `data`, then calls `onFind` on them. I think in this case we can use `unknown` and cast into `Data` before calling `onFind`